using-mobile-extensions/mobile-core/mobile-core-release-notes.md

```diff
@@ -7,14 +7,11 @@ The following updates were made in this release:
 **Android Core 1.5.0**
 
 * Fixed a cursor leak.
-
 * Fixed an issue where the advertising identifier was duplicated in the response to the `MobileCore.getSDKIdentifiers` API.
-
 * Added support for overriding internal network stack with customer-provided code.
+* Added a new interface with failure callback, `AdobeCallbackWithError`, which can be used with the `MobileCore.getPrivacyStatus`,  `MobileCore.getSdkIdentities ` methods. 
 
-* Added new a interface with failure callback, `AdobeCallbackWithError`, which can be used with the methods `MobileCore.getPrivacyStatus`,  `MobileCore.getSdkIdentities`. And we will gradually add the support in other extensions to enable the failure callback.
-
-  
+​       We plan to gradually add the ability to enable failure callback to the other extensions.
```

Comment:
What I meant was this:
"* Added a new interface with failure callback, AdobeCallbackWithError, which can be used with the MobileCore.getPrivacyStatus, MobileCore.getSdkIdentities methods.
​     We plan to gradually add the ability to enable failure callback to the other extensions."
where "We" should line up with "Added". Hope that is clearer!